js/src/setup-mc/setup-stepper/setup-free-listings/index.js

```diff
@@ -41,7 +42,8 @@ const SetupFreeListings = ( props ) => {
 	const {
 		data: finalCountryCodesData,
 	} = useTargetAudienceFinalCountryCodes();
-	const { saveSettings, saveShippingRates } = useAppDispatch();
+	const { saveSettings } = useAppDispatch();
+	const { saveShippingRates } = useSaveShippingRates();
```

Comment:
Actually I think that is fine. In case for some reasons the `onContinue` doesn't go to the next page, having the button displayed there with loading state might look like a bug.

Anyway, I'm fine with moving it into the catch block too (because `onContinue` does go to the next page), so I have moved it there in [`43513b9` (#1313)](https://github.com/woocommerce/google-listings-and-ads/pull/1313/commits/43513b9dd139c8aecd30bffc01f0a6caf2c10bad).